Docs/troubleshooting.md

```diff
@@ -0,0 +1,236 @@
+
+
+#Troubleshooting
+Provide some tips and tricks to help customize and troubleshoot the solution.
+
+## Using a Proxy Server to connect your Concentrator to Azure
+
+This is an optional configuration that should only be executed if your concentrator needs to use a proxy server to communicate with Azure.
+
+Follow [this guide](/Docs/devguide.md#use-a-proxy-server-to-connect-your-concentrator-to-azure) to:
+
+1. Configure the Docker daemon and the IoT Edge daemon on your device to use a proxy server.
+2. Configure the `edgeAgent` properties in the `config.yaml` file on your device.
+3. Set environment variables for the IoT Edge runtime in the deployment manifest.
+4. Add the `https_proxy` environment variable to the `LoRaWanNetworkSrvModule` in IoT Hub.
+
+## LoRa Device provisioning
+
+A LoRa device is a normal IoT Hub device with some specific device twin tags. You manage it like you would with any other IoT Hub device.
+**To avoid caching issues you should not allow the device to join or send data before it is provisioned in IoT Hub. In case that you did plese follow the ClearCache procedure that you find below.**
+
+### ABP (personalization) and OTAA (over the air) provisioning
+
+- Login in to the Azure portal go to IoT Hub -> IoT devices -> Add
```

Comment:
add "," after the words Azure Portal... 

Login in to the Azure portal, go to IoT Hub -> IoT devices -> Add